apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -51,12 +52,11 @@ type BookingFormValues = {
   };
 };
 
-const BookingPage = (props: BookingPageProps) => {
+const BookingPage = ({ eventType, booking, profile }: BookingPageProps) => {
   const { t, i18n } = useLocale();
   const router = useRouter();
   const { contracts } = useContracts();
-  const { eventType } = props;
-
+  const { data: session } = useSession();
```

Comment:
Here we call the new useSession hook, which when authenticated provides us with user + name + email